content/docs/studio/troubleshooting.md

```diff
@@ -359,29 +358,11 @@ For a model to be displayed in the model registry, it has to be registered using
 [GTO]. You can [register the model] from Iterative Studio or with the [`gto`
 CLI].
 
-## How can I remove models from my model registry
-
-To remove a model from the model registry, use the `Deprecate model` menu item
-in the 3-dot menu next to the model name. You can also remove all of a project's
-models by deleting the project from your projects dashboard.
-
 ## My models have disappeared even though I did not remove (deprecate) them
 
-Models can also be removed by deleting the associated project from the projects
-dashboard. So make sure that the project is not deleted.
-
-## How can I un-assign stages from model versions
-
-To un-assign any stage from a model, open the corresponding Git repository (in
-GitHub, GitLab or Bitbucket). Then, remove the Git tag pertaining to the stage
-that you want to un-assign.
-
-After removing the Git tag, reparse the repository. To do this, open the project
-in Iterative Studio and then click on `Force import` as shown below.
-
-![Showing and hiding columns](https://static.iterative.ai/img/studio/force_import.gif)
-
-Currently, you cannot un-assign stages from the Iterative Studio user interface.
+When a project is deleted from the projects dashboard, all its models get
+automatically removed form the model registry. So make sure that the project is
+not deleted.
```

Comment:
It sounds more dangerous to me than it really is. If they re-add the project, everything will reappear without losing any info, right? Should we clarify this and make clear that the git tags won't disappear, so everything is safe as long as the git remote is intact?

Reply:
You're right. I've updated it [here](https://github.com/iterative/dvc.org/pull/4638/commits/e2e893cbe00d960669980e0c9cd8a835628f3ea4).